src/views/assets/Domain/components/AddGatewayDialog.vue

```diff
@@ -73,7 +74,7 @@ export default {
         cleanFormValue(values) {
           const data = []
           values.gateways.forEach(item => {
-            const d = { id: item.pk, domain: this.$route.params.id }
+            const d = { id: item.pk, domain: vm.object.id }
             data.push(d)
           })
           return data
```

Comment:
None of the provided code is present to analyze. Please provide an actual piece of JavaScript or React component that needs analysis.